sst/mtm1m3/Hardpoint_Recent_History.ipynb

```diff
@@ -86,42 +106,41 @@
    "source": [
     "%matplotlib inline\n",
     "\n",
-    "states = [['Pos',3,4], ['Neg',4,5]]\n",
+    "states = [['Pos',2,3,4], ['Neg',3,4,5]]\n",
```

Comment:
I would consider putting a comment here, listing the enumeration to make it more interoperable
e.g 
# Pos: MovingNegative -> TestingPositive -> TestingNegative
# Neg: TestingPositive -> TestingNegative -> MovingReference